tests/testthat/test-config.R

```diff
@@ -171,7 +204,66 @@ test_that("prevalence graph config containe valid intervention ids", {
   IRS <- ids[[4]]
   expect_equal(IRS, mint_intervention(0, 1, "pto"))
   ITN_IRS <- ids[[5]]
-  expect_equal(ITN_IRS,mint_intervention(1, 1, "std"))
+  expect_equal(ITN_IRS, mint_intervention(1, 1, "std"))
```

Comment:
Opened mrc-1975 to set up linting to help with this churn

Reply:
nice, thanks